SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -38,9 +38,12 @@ To get access to the SpatialOS Unreal Engine fork, you need to link your GitHub
 
 ### Step 3: Add a new SSH key to your GitHub account
 
-You must add an SSH key to your GitHub account in order to automatically download the GDK repositories as part of this setup step.
+If you have not already configured your GitHub account to use an SSH key, you must do so in order to automatically download the GDK repositories using **InstallGDK.bat** as part of the next setup step.
```

Comment:
```suggestion
If you have not already configured your GitHub account to use an SSH key, you must do this in order to automatically download the GDK repositories as part of the next setup step.
```
Suggest "this" is less formal register.  Although "make it so" should be in the docs somewhere, IMHO.
Suggest we lose the batch file name as you don't need to know this yet. Cognitive overload during _Get started_.